Makefile

```diff
@@ -160,6 +158,9 @@ ASM_SOURCES := $(shell find . -name \*.S)
 ASM_OFFSETS := $(shell find . -name $(ASM_OFFSETS_C))
 LINK_SCRIPT := $(shell find . -name \*.ld)
 
+ASM_OFFSETS_H := $(shell for f in $(ASM_OFFSETS); do echo include/$${f%.c}.h; done)
+ASM_OFFSETS_S := $(shell for f in $(ASM_OFFSETS); do echo $${f%.c}.s; done)
+
```

Comment:
No need for the shell, this can be done in GNU make alone:

```make
ASM_OFFSETS_S := $(ASM_OFFSETS:.c=.s)
ASM_OFFSETS_H := $(addprefix include/,$(ASM_OFFSETS:.c=.h))
```

Reply:
Ha, nice! Fixed and thanks :-)